sdk_extensions/zpages/README.md

```diff
@@ -98,4 +98,123 @@ details. For example, here are the details of the `ChildSpan` latency sample (ro
 The /traceconfigz zPage displays information about the currently active tracing configuration and 
 provides an interface for users to modify relevant parameters. Here is what the web page looks like:
 
-![traceconfigz](img/traceconfigz.png)
\ No newline at end of file
+![traceconfigz](img/traceconfigz.png)
+
+## Benchmark Testing
+
+This module contains two sets of benchmark tests: one for adding spans to an instance of
+TracezSpanBuckets and another for retrieving counts and spans with TracezDataAggregator. You can run
+the tests yourself with the following commands:
+
+```
+./gradlew -PjmhIncludeSingleClass=TracezSpanBucketsBenchmark clean :opentelemetry-sdk-extension-zpages:jmh
+./gradlew -PjmhIncludeSingleClass=TracezDataAggregatorBenchmark clean :opentelemetry-sdk-extension-zpages:jmh
+```
+
+The expected results for `TracezSpanBucketsBenchmark` are as follows:
```

Comment:
I would add a sentence that describes the hardware (CPU/RAM/OS) used to obtain these results.

Reply:
Yeah I'd personally leave these out of READMEs and paste them in PR to provide some history along with code changes. Especially, it's too hard to repro even with specs about the CPU (except if using a cloud VM) that we couldn't call them expected results.

Having JMH to measure future code changes in a relative way is great.

Reply:
I agree. Leave them out of the README and keep them in the PR description, along with some indication of JVM version and hardware that they were run on.

Reply:
Updated the top level comment.